### PR TITLE
🐛Fix IE11 with a basic polyfill

### LINKED ISSUE #18184: Date picker does not open in IE 11
## What's the issue?

The date picker does not open in Internet Explorer 11. The following error is thrown in the console:

SCRIPT438: Object doesn't support property or method 'from'
File: v0.js, Line: 10, Column: 275

## How do we reproduce the issue?

If this is a bug please provide a public URL and ideally a reduced test case (e.g. on jsbin.com) that exhibits only your issue and nothing else.  Then provide step-by-step instructions for reproducing the issue:

1. Navigate to https://ampbyexample.com/components/amp-date-picker
2. Scroll down to the "Single Date Picker" section
3. Click the date input element or one of the action buttons

The above error is thrown in the console and the date picker does not open.

## What browsers are affected?

Internet Explorer 11.0.9600.19002

## Which AMP version is affected?

Version 1809142227020

### PR DESCRIPTION
**Pull request by cvializ**
Fixes #18184 with a partial polyfill instead of the full polyfill from #18879 

The partial polyfill is OK because the `Array.from` call in `_toConsumableArray` (generated from the spread operator usages) is only ever called with Arrays or `false`:

https://github.com/airbnb/react-dates/blob/f48b820449b2a460761ead4c056ccfac2322bb78/src/components/DayPickerNavigation.jsx#L109-L123

### CODE REVIEW

File: third_party/react-dates/index.js [hunk 1]
```diff
@@ -1,2 +1,18 @@
...
+  Array.from = function (arrayLike) {
+    if (!arrayLike) {
+      return [];
+    }
+
+    if (!arrayLike.length) {
+      return [];
+    }
+
+    return [].concat(arrayLike);
```
Comment (jridgewell) [inline third_party/react-dates/index.js]: Might be better to do a for loop? Concat will only spread arrays, not array-likes.

File: third_party/react-dates/index.js [hunk 2]
```diff
@@ -1,2 +1,18 @@
 import 'react-dates/initialize';
 import 'react-dates';
+
+// A very basic polyfill for Array.from,
+// to avoid needing to use a full 3p implementation
+if (!Array.from) {
+  Array.from = function (arrayLike) {
```
Comment (jridgewell) [inline third_party/react-dates/index.js]:
```js
Object.defineProperty(Array, 'from', {
  writable: true,
  enumerable: false,
  configurable: true,
  value: poly,
});

**Comment by cvializ** [comment]
@jridgewell PTAL

**Comment by cvializ** [comment]
Thank you for your review!

**Pull request merged**